Extensions/TwitterSentimentAnalysis/Src/vss-extension.json

```diff
@@ -47,6 +47,11 @@
 			"path": "readme.md"
 		}
 	},
+	"links": {
+		"license": {
+			"uri": "https://go.microsoft.com/fwlink/?linkid=870392"
```

Comment:
should this URL point to
https://raw.githubusercontent.com/Microsoft/vsts-rm-extensions/master/LICENSE